### PR TITLE
Cleanup

### LINKED ISSUE #49: Clean up old code (as in remove)

### PR DESCRIPTION
**Pull request by conniec218**
Closes #49

### CODE REVIEW

File: web/src/view/nav/NavBar.tsx [hunk 1]
```diff
@@ -12,27 +12,19 @@ import { getPath, getWelcomePath, Route } from './route'
...
-    name: 'map',
-    path: getPath(Route.MAP),
-  },
-]
+// const otherTabs = [
+//   {
+//     name: 'map',
+//     path: getPath(Route.MAP),
+//   },
+// ]
```
Comment (Timthetic) [inline web/src/view/nav/NavBar.tsx]: Why add this?

File: web/src/view/nav/NavBar.tsx [hunk 2]
```diff
@@ -12,27 +12,19 @@ import { getPath, getWelcomePath, Route } from './route'
...
 
 export function NavBar() {
-  const location = useLocation()
+  // const location = useLocation()
```
Comment (Timthetic) [inline web/src/view/nav/NavBar.tsx]: Stahp

File: web/src/view/nav/NavBar.tsx [hunk 3]
```diff
@@ -55,7 +47,7 @@ export function NavBar() {
     return void 0
   }, [toast])
 
-  const tabs = isSmall ? [otherTabs.find(t => location.pathname.startsWith(t.path)) || otherTabs[0]] : otherTabs
+  // const tabs = isSmall ? [otherTabs.find(t => location.pathname.startsWith(t.path)) || otherTabs[0]] : otherTabs
```
Comment (Timthetic) [inline web/src/view/nav/NavBar.tsx]: stahpet

File: web/src/view/nav/NavBar.tsx [hunk 4]
```diff
@@ -69,9 +61,9 @@ export function NavBar() {
           {isSmall && <div style={{ flex: 1 }} />}
 
           {/* layout additional tabs (possibly hidden for small screens) */}
-          {tabs.map((tab, i) => (
+          {/* {tabs.map((tab, i) => (
             <NavItem key={i} {...tab} />
-          ))}
+          ))} */}
```
Comment (Timthetic) [inline web/src/view/nav/NavBar.tsx]: STAHPET

File: web/src/view/nav/NavBar.tsx [hunk 5]
```diff
@@ -93,9 +85,9 @@ function NavMenu(props: { show: boolean; onClick: () => void }) {
       {props.show && (
         <Modal>
           <NavMenuModal>
-            {otherTabs.map((tab, i) => (
+            {/* {otherTabs.map((tab, i) => (
               <NavItem key={i} {...tab} />
-            ))}
+            ))} */}
```
Comment (Timthetic) [inline web/src/view/nav/NavBar.tsx]: SCREEECCHH

**Review by Timthetic — APPROVED** [review]
I skeem 👀. I skeem and cohde👩‍💻 looks gud✅. I skeem cahments 💬 and dohnt ❌ like cahments een cohde👩‍💻. Make cahments 💬 go away and I approohv 🚀

Also, I'll make a special note that this PR will **require that we all clear our local databases** once it is merged.

**Comment by conniec218** [comment]
hopefully I have alleviated your pain and suffering here

**Pull request merged**